apps/web/src/pages/Skills/components/SkillTable.tsx

```diff
@@ -49,110 +29,108 @@ interface SkillTableProps {
 
 export const SkillTable = ({ skills, title }: SkillTableProps) => {
   const intl = useIntl();
-  const locale = getLocale(intl);
   const paths = useRoutes();
-  const columns = useMemo<ColumnsOf<Skill>>(
-    () => [
-      {
-        Header: intl.formatMessage({
-          defaultMessage: "ID",
-          id: "Z6o8ym",
-          description: "Title displayed on the Skill table ID column.",
-        }),
-        accessor: "id",
+  const columns = [
+    columnHelper.accessor("id", {
+      id: "id",
+      enableColumnFilter: false,
+      header: intl.formatMessage(adminMessages.id),
+    }),
+    columnHelper.accessor((skill) => getLocalizedName(skill.name, intl), {
+      id: "name",
+      header: intl.formatMessage({
+        defaultMessage: "Name",
+        id: "BOeBpE",
+        description: "Title displayed for the skill table Name column.",
+      }),
+      meta: {
+        isRowTitle: true,
       },
+    }),
+    columnHelper.accessor(
+      (skill) => getLocalizedName(skill.description, intl, true),
       {
-        Header: intl.formatMessage({
-          defaultMessage: "Name",
-          id: "BOeBpE",
-          description: "Title displayed for the skill table Name column.",
-        }),
-        accessor: (skill) => skill.name?.[locale],
-      },
-      {
-        Header: intl.formatMessage({
+        id: "description",
+        header: intl.formatMessage({
           defaultMessage: "Description",
           id: "9yGJ6k",
           description:
             "Title displayed for the skill table Description column.",
         }),
-        accessor: (skill) => skill.description?.[locale],
       },
-      {
-        Header: intl.formatMessage({
-          defaultMessage: "Keywords",
-          id: "I7rxxQ",
-          description: "Title displayed for the skill table Keywords column.",
-        }),
-        // keywords[locale] throws type problems
-        accessor: (skill) => {
-          if (locale === "en") {
-            if (skill.keywords && skill.keywords.en)
-              return skill.keywords.en.join(", ");
-          }
-          if (locale === "fr") {
-            if (skill.keywords && skill.keywords.fr)
-              return skill.keywords.fr.join(", ");
-          }
-          return "";
-        },
+    ),
+    columnHelper.accessor((skill) => keywordsAccessor(skill, intl), {
+      id: "keywords",
+      header: intl.formatMessage({
+        defaultMessage: "Keywords",
+        id: "I7rxxQ",
+        description: "Title displayed for the skill table Keywords column.",
+      }),
+    }),
+    columnHelper.accessor((skill) => familiesAccessor(skill, intl), {
+      id: "skillFamilies",
+      header: intl.formatMessage(adminMessages.skillFamilies),
+      cell: ({ row: { original: skill } }) =>
+        skillFamiliesCell(skill.families, intl),
+    }),
```

Comment:
Given a skill can belong to many families, the sort on this bit won't be reliable always
Unless this is somehow sorting pill-by-pill for a cell and I've failed to notice

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/55f81309-a582-48c9-9b47-a2e815749257)

Reply:
Oops, looks like we were sorting the accessor but not the cell so it looked odd! Should be fixed in [sort skill family cell](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8020/commits/1b834d990507a31dbb50d0e5aaa79213eb9163d5)